resources/kcp/charts/kyma-metrics-collector/templates/service.yaml

```diff
@@ -12,6 +12,9 @@ spec:
     port: {{ .Values.service.port }}
     protocol: {{ .Values.service.protocol }}
     targetPort: {{ .Values.config.portName }}
+  - port: {{ .Values.global.istio.proxy.port }}
+    protocol: TCP
+    name: proxy-status
```

Comment:
Not a review, but just a question. Do the port names in istio need to follow a specific pattern?
https://istio.io/v1.1/docs/setup/kubernetes/prepare/requirements/

Reply:
I have updated the port name according to the Istio naming convention.